src/control/cmd/daos_server/storage_nvme.go

```diff
@@ -253,11 +256,13 @@ func (cmd *prepareNVMeCmd) Execute(_ []string) error {
 }
 
 type resetNVMeCmd struct {
-	nvmeCmd
-	PCIBlockList string `long:"pci-block-list" description:"Comma-separated list of PCI devices (by address) to be ignored when unbinding devices from Kernel driver to be used with SPDK (default is no PCI devices)"`
-	TargetUser   string `short:"u" long:"target-user" description:"User that will own hugepage mountpoint directory and vfio groups."`
-	DisableVFIO  bool   `long:"disable-vfio" description:"Force SPDK to use the UIO driver for NVMe device access"`
-	Args         struct {
+	cmdutil.LogCmd  `json:"-"`
+	helperLogCmd    `json:"-"`
+	iommuCheckerCmd `json:"-"`
```

Comment:
Are the json annotations needed? I don't think we serialize command structs to json anywhere.